docs: added flutter version requirement tips for building on linux (#43)
AppFlowy-IO/AppFlowy-Docs: essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md

![img.png](../../../../.gitbook/assets/build\_step\_two.png)

{% hint style="warning" %}

> Flutter 3.0.5 is the version that AppFlowy is built and tested now. If you installed the Flutter before, make sure the version is correct. You can get the the Flutter version by the command as below.

Reviewer: I think that the following change would help with clarity:
```suggestion
Flutter version 3.0.5 is the version that AppFlowy is built and tested on. If you installed Flutter before, make sure the version matches AppFlowy's. You can get the the Flutter version using this command:
```

Author: Ok, I will make changes
